src/mongo/initDb.js

```diff
@@ -1,16 +1,14 @@
 const mongoose = require('mongoose');
+const databaseManager = require('./databaseManager');
 
 const databaseInit = () => {
-    mongoose.connect('mongodb://localhost:27017/', {
-        useNewUrlParser: true,
-        dbName: "portafolio",
-        connectTimeoutMS: 5000
-    });
+    const mongoDatabase = new databaseManager(
+        mongoose,
+        'mongodb://localhost:27017/',
```

Comment:
I think mongo Atlas could be a good option to start something instead of having it as local mongo, or having to set it up on server... just something to think about for a quick start! :)

Reply:
Will move to Atlas later.  It will definitely make things easier when deploying